### PR TITLE
feat: argument builder for entity type

### PR DESCRIPTION
**Pull request by arjunlalb**

### CODE REVIEW

File: projects/distributed-tracing/src/shared/graphql/request/builders/argument/graphql-argument-builder.ts [hunk 1]
```diff
@@ -73,4 +74,8 @@ export class GraphQlArgumentBuilder {
       value: new GraphQlEnumArgument(type)
     };
   }
+
+  public forEntityType(type: EntityType): GraphQlArgument {
```
Comment (skjindal93) [inline projects/distributed-tracing/src/shared/graphql/request/builders/argument/graphql-argument-builder.ts:78]: Can we add a small unit test for this?

**Review by skjindal93 — APPROVED** [review]

File: projects/distributed-tracing/src/shared/graphql/request/builders/argument/graphql-argument-builder.ts (hunk 1 above)
Comment (arjunlalb) [reply projects/distributed-tracing/src/shared/graphql/request/builders/argument/graphql-argument-builder.ts:78]: Added a whole lot of tests :)

File: projects/distributed-tracing/src/shared/graphql/request/builders/argument/graphql-argument-builder.test.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import {
+  GraphQlArgumentBuilder,
+  GraphQlFieldFilter,
+  GraphQlOperatorType,
+  GraphQlTimeRange,
+  SpecificationBuilder
+} from '@hypertrace/distributed-tracing';
+import { GraphQlEnumArgument } from '@hypertrace/graphql-client';
+
+describe('Graphql Specification Builder', () => {
```
Comment (skjindal93) [inline projects/distributed-tracing/src/shared/graphql/request/builders/argument/graphql-argument-builder.test.ts:10]: Graphql Argument Builder

File: projects/distributed-tracing/src/shared/graphql/request/builders/argument/graphql-argument-builder.test.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import {
```
Comment (skjindal93) [inline projects/distributed-tracing/src/shared/graphql/request/builders/argument/graphql-argument-builder.test.ts:1]: Thaaaaaanks for adding the tests

File: projects/distributed-tracing/src/shared/graphql/request/builders/argument/graphql-argument-builder.test.ts (hunk 2 above)
Comment (arjunlalb) [reply projects/distributed-tracing/src/shared/graphql/request/builders/argument/graphql-argument-builder.test.ts]: done

**Review by skjindal93 — APPROVED** [review]

**Pull request merged**